src/components/Home/Main/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { DealsCard } from "../common/deals-card"
+import {Banner } from "../common/main-banner"
+
+export function Main() {
+    return (
+        <main className="main-container">
+            {/* main banner */}
+            <Banner/>
+            {/* main feature-section */}
+            <section className="main-container featured-section">
+                <h1>Featured Categories</h1>
+                <div className="feature-section-info">
+                    <li className="feature-list">
+                        <img className="feature-img" src="/assets/shoes.png" alt="" />
```

Comment:
In react, you need to import the images in order to show them on screen. Is this working?

Reply:
Yes it works if its in the public folder.

Reply:
Oh! Didn't know that